packages/pds/src/mailer/index.ts

```diff
@@ -51,6 +51,19 @@ export class ServerMailer {
     })
   }
 
+  // TODO: Implement gmail sending here?
```

Comment:
I think this TODO may be resolved :)

Reply:
Or maybe the method is no longer used?

Reply:
yep no longer used, leftover from early attempt at implementation.